### PR TITLE
CASMTRIAGE-3246: fixed generate_switch_config.md, generating switch config for single …

### PR DESCRIPTION
**Pull request by kosonenj**
## Summary and Scope

Fixed generate_switch_config.md, generating switch config for single device was incorrect. 

Essentially for single device, the config file will just be created in working directory vs in generate whole network will create a folder to output generated config files.

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves [CASMTRIAGE-3246](https://jira-pro.its.hpecorp.net:8443/browse/CASMTRIAGE-3246)
* Change will also be needed in `1.2.5, done in branch CASMTRIAGE-3246-1`

## Testing

Re-run command

### Tested on:

  * `Gamora`
  * Local development environment=

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [ ] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

**Review by lukebates123 — APPROVED** [review]

**Review by gbaker-hpe — APPROVED** [review]

File: operations/network/management_network/generate_switch_configs.md [hunk 1]
```diff
@@ -27,16 +27,25 @@ The following are the different architectures that can be specified:
 * ***Full*** – Aruba-based Leaf-Spine systems, usually customer production systems.
 * ***V1*** – Dell and Mellanox based systems of either a TDS or Full layout.
 
-Select one of the following commands to generate a configuration file for a single switch, or for the full system.
+Generating a configuration file can be done for a single switch, or for the full system. In below, example commands for both scenarios:
```
Comment (mharding-hpe) [inline operations/network/management_network/generate_switch_configs.md:30]:
```suggestion
Generating a configuration file can be done for a single switch, or for the full system. Below are example commands for both scenarios:
```

File: operations/network/management_network/generate_switch_configs.md [hunk 2]
```diff
@@ -27,16 +27,25 @@ The following are the different architectures that can be specified:
 * ***Full*** – Aruba-based Leaf-Spine systems, usually customer production systems.
 * ***V1*** – Dell and Mellanox based systems of either a TDS or Full layout.
 
-Select one of the following commands to generate a configuration file for a single switch, or for the full system.
+Generating a configuration file can be done for a single switch, or for the full system. In below, example commands for both scenarios: 
+
+***Important***
+
+You will need to modify following items from your command:
```
Comment (mharding-hpe) [inline operations/network/management_network/generate_switch_configs.md:34]:
```suggestion
Modify the following items in your command:
```

File: operations/network/management_network/generate_switch_configs.md [hunk 3]
```diff
@@ -27,16 +27,25 @@ The following are the different architectures that can be specified:
...
 * ***V1*** – Dell and Mellanox based systems of either a TDS or Full layout.
 
-Select one of the following commands to generate a configuration file for a single switch, or for the full system.
+Generating a configuration file can be done for a single switch, or for the full system. In below, example commands for both scenarios: 
+
+***Important***
+
+You will need to modify following items from your command: 
+
+* --csm : Which CSM version configuration do you want to use. I.e.  1.2 or 1.0
```
Comment (mharding-hpe) [inline operations/network/management_network/generate_switch_configs.md:36]:
```suggestion
* `--csm` : Which CSM version configuration do you want to use? For example, 1.2 or 1.0
```

File: operations/network/management_network/generate_switch_configs.md [hunk 4]
```diff
@@ -27,16 +27,25 @@ The following are the different architectures that can be specified:
...
+* --a   : Which architecture is the system based of. (See above)
```
Comment (mharding-hpe) [inline operations/network/management_network/generate_switch_configs.md:37]:
```suggestion
* `--a`   : What is the system architecture? (See above)
```

File: operations/network/management_network/generate_switch_configs.md [hunk 5]
```diff
@@ -27,16 +27,25 @@ The following are the different architectures that can be specified:
...
+* --ccj : Match the ccj.json file to the one you created for your system.
```
Comment (mharding-hpe) [inline operations/network/management_network/generate_switch_configs.md:38]:
```suggestion
* `--ccj` : Match the `ccj.json` file to the one you created for your system.
```

File: operations/network/management_network/generate_switch_configs.md [hunk 6]
```diff
@@ -27,16 +27,25 @@ The following are the different architectures that can be specified:
...
+* --sls : Match the sls_file.json to the one you created for your system.
```
Comment (mharding-hpe) [inline operations/network/management_network/generate_switch_configs.md:39]:
```suggestion
* `--sls` : Match the `sls_file.json` to the one you created for your system.
```

**Review by mharding-hpe — CHANGES_REQUESTED** [review]

**Comment by mharding-hpe** [comment]
We also need this in the main branch (ideally the primary PR would be to main and we would backport it to the previous releases, but we can do it the other way)

**Pull request merged**